File: FRONT_END_CAR/src/app/Shared/models/brand.ts
export class Brand {
    brandId: string;
    brandName: string;
    creationDate: string;
    updateDate: string | null;
  
    constructor(data: any) {
      this.brandId = data.brandId || '';
      this.brandName = data.brandName || '';
      this.creationDate = data.creationDate || '';
      this.updateDate = data.updateDate || null;
    }
  }
  

File: BACK_END_CAR/CAR_RENTAL_API/Entities/Brand.cs
namespace CAR_RENTAL_API.Entities
{
    public class Brand
    {
        public Guid Id { get; set; } = Guid.NewGuid();
        public string BrandName { get; set; }
        public DateTime CreationDate { get; set; } = DateTime.UtcNow;
        public DateTime? UpdateDate { get; set; }

        public ICollection<Car> Cars { get; set; }
    }
}
